packages/dev/core/src/AudioV2/abstractAudio/subNodes/audioSubNode.ts

```diff
@@ -0,0 +1,9 @@
+/** @internal */
+export class _AudioSubNode {
```

Comment:
Doesn't need to be a class (unless you are extending it?). Can be a const object with 3 keys

Reply:
Or a const enum?